src/components/forms/JurisdictionMetadataDownload/index.tsx

```diff
@@ -0,0 +1,205 @@
+  key: string;
+  value: string;
+  label: string;
+  description?: string;
+  uuid: string;
+  settingsId: string;
+  settingIdentifier: string;
+  settingMetadataId: string;
+  teamId?: string;
+  providerId?: string;
+  locationId?: string;
+  v1Settings: boolean;
+  resolveSettings: boolean;
+  documentId: string;
+  serverVersion: number;
+  type: string;
+}
+
+/** interface for each select dropdown option */
+export interface Option {
+  label: string;
+  value: string;
+}
+
+export interface JurisdictionMetadataDownloadFormFields {
+  identifier: Option;
+}
+
+export interface JurisdictionMetadataDownloadFormProps {
+  disabledFields: string[];
+  serviceClass: OpenSRPService;
+  submitForm: (
+    setSubmitting: (isSubmitting: boolean) => void,
+    setGlobalError: (errorMessage: string) => void,
+    props: JurisdictionMetadataDownloadFormProps,
+    values: JurisdictionMetadataDownloadFormFields
+  ) => void;
+  initialValues: JurisdictionMetadataDownloadFormFields;
+}
+
+export const defaultInitialValues: JurisdictionMetadataDownloadFormFields = {
+  identifier: { label: '', value: '' },
+};
+
+const createCsv = (entries: JurisdictionMetadataFile[]): void => {
+  const csv: string = Papaparse.unparse(entries, {
+    header: true,
+  });
+  // download file
+  download(csv, JURISDICTION_METADATA, 'text/csv');
+};
+
+const downloadFile = (response: JurisdictionMetadataResponse[]) => {
+  const entries: JurisdictionMetadataFile[] = [];
+  response.forEach(item => {
+    const metaType = item.settingIdentifier.replace('jurisdiction_metadata-', '');
+    const entry: JurisdictionMetadataFile = {
+      jurisdiction_id: item.key,
+      jurisdiction_name: item.label,
+      [metaType]: item.value,
+    };
+    entries.push(entry);
+  });
+  createCsv(entries);
+};
+
+export const submitForm = (
+  setSubmitting: (isSubmitting: boolean) => void,
+  setGlobalError: (errorMessage: string) => void,
+  props: JurisdictionMetadataDownloadFormProps,
+  values: JurisdictionMetadataDownloadFormFields
+) => {
+  const { serviceClass } = props;
+  const params = {
+    identifier: values.identifier.value,
+    serverVersion: 0,
+  };
+  serviceClass
+    .list(params)
+    .then((response: JurisdictionMetadataResponse[]) => {
+      downloadFile(response);
+      growl(FILE_DOWNLOADED_SUCCESSFULLY, {
+        onClose: () => setSubmitting(false),
+        type: toast.TYPE.SUCCESS,
+      });
+    })
+    .catch((e: Error) => {
+      setGlobalError(e.message);
+      setSubmitting(false);
+    });
+};
+
+const JurisdictionMetadataDownloadForm = (props: JurisdictionMetadataDownloadFormProps) => {
+  const { initialValues } = props;
+  const [globalError, setGlobalError] = useState<string>('');
+  const [disabled, setDisabled] = useState<boolean>(true);
+  const identifierOptions = [
+    { value: JURISDICTION_METADATA_RISK, label: 'Risk' },
+    { value: JURISDICTION_METADATA_COVERAGE, label: 'Coverage' },
+  ];
```

Comment:
Ideally we want to populate this based on the metadata that has been previously uploaded to the server.  I think for tomorrow's release doing this is fine, but please create an issue to make this dynamic

Reply:
Okay. I'm creating one